lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -674,6 +694,9 @@ void RawMemManager::onFunctionEntry(const Function &fn) {
 Expr RawMemManager::zeroedMemory() const {
   return m_memRepr->FilledMemory(ptrSort(), m_ctx.alu().si(0, wordSzInBits()));
 }
-
+const std::unique_ptr<OpSemAllocator> &RawMemManager::getMAllocator() const {
```

Comment:
probably wnat to return `OpSemAllocator&` rather than ref to unique pointer

Reply:
done